addOns/client/src/main/java/org/zaproxy/addon/client/ExtensionClientIntegration.java

```diff
@@ -318,6 +309,16 @@ public void eventReceived(Event event) {
                         eventConsumer, "org.zaproxy.zap.extension.spiderAjax.SpiderEventPublisher");
     }
 
+    private ClientMap getClientTree() {
+        clientTree =
+                new ClientMap(
+                        new ClientNode(
+                                new ClientSideDetails(
+                                        Constant.messages.getString("client.tree.title"), null),
+                                this.getModel().getSession()));
```

Comment:
Do we really want this instantiating new objects every time it's called?

Reply:
Yeah, thats a bug - fixed :)